### PR TITLE
Fixes for release 0623

### LINKED ISSUE #25: Efficiency scores are not shown if `dea.slack` returns `NULL`
In some circumstances, the function `dea.slack` can return a null-value. This will break the app with the error message `Error in if: missing value where TRUE/FALSE needed?`. The bug is not reproducible for all files, so a fix can be difficult to find. A more simple approach would be to catch a null-value and prompt the user with a warning that slack values could not be estimated.

### PR DESCRIPTION
**Pull request by ohjakobsen**
This PR closes #25 and #30 and introduce various other fixes:

* List wise deletion is performed if the user selects input and/or output variables where there are incomplete cases
* For VRS, the frontier line could dip if one or more observations on the far right side on the x-axis has a value lower than the maximum value on the y-axis. This problem has been resolved, and frontier line for NIRS has been added as well
* General UI improvements
* Bumped version to 0.1.0

### CODE REVIEW

File: inst/app/conditionalUI.R
Comment (lse81) [inline inst/app/conditionalUI.R:1]: Looks ok.

File: inst/app/server.R
Comment (lse81) [inline inst/app/server.R:1]: There is some problems with the listwise deletion function.  Listwise deletion was not performed  even though I had some missing observation in my dataset.

Comment (lse81) [inline inst/app/server.R:1]: The slack-values were all 0 when using an output-oriented model. It should be slack values differenct from 0 for both input and output-oriented models.

**Review by lse81 — CHANGES_REQUESTED** [review]
This looks very good. I like the listwise deletion function. However i still get some errors we have to look into. 
1) Error message  "could not find function "card"" still present when open the app from R-studio. 
2) Listwise deletion was not performed neither with .csv-file nor the .dta-file. 
3) There are some problems with the encoding. With UTF8 we do not get Æ, Ø, Å.  
4) I get an error message when specifying the model with either single input/output or multiple inputs/outputs. "Error: Y is not a numeric matrix (or numeric data.frame)"
'Y' is not a numeric matrix (or numeric data.fram

**Comment by ohjakobsen** [comment]
> This looks very good. I like the listwise deletion function. However i still get some errors we have to look into.
> 
> 1. Error message  "could not find function "card"" still present when open the app from R-studio.
> 2. Listwise deletion was not performed neither with .csv-file nor the .dta-file.
> 3. There are some problems with the encoding. With UTF8 we do not get Æ, Ø, Å.
> 4. I get an error message when specifying the model with either single input/output or multiple inputs/outputs. "Error: Y is not a numeric matrix (or numeric data.frame)"
>    'Y' is not a numeric matrix (or numeric data.fram

Issue 1 and 2 is probably an issue with your local install. Encoding should be supported, but this might be a Windows issue that we need to look into. The last error message is a bit more cryptic and needs to be explored. I will need a step by step guide to reproduce the issue.

File: inst/app/server.R
Comment (ohjakobsen) [reply inst/app/server.R:1]: This was an issue with the commit history. Should be fixed on the latest commit.

Comment (ohjakobsen) [reply inst/app/server.R:1]: We need to look at this in a separate PR. We can probably do this in relation to #30.

**Review by lse81 — APPROVED** [review]
Looks good!

**Pull request merged**